CHANGELOG.md

```diff
@@ -1,5 +1,88 @@
 # Changelog
 
+## v0.1.54
+
+<!-- generated comparing v0.1.53..master -->
+
+_Apr 12, 2024_
```

Comment:
Sorry for the late review, but looks like this is doubled in the changelog :D

Reply:
Ah damn! Thanks, I'll fix it now